test/fixtures/cross_agent_tests/README.md

```diff
@@ -17,7 +17,7 @@ Push access to this repository is granted via membership in the cross-agent-team
 | [rum_footer_insertion_location](rum_footer_insertion_location) | Describe where the RUM footer (aka "client config") should be inserted.  These tests do not apply to agents which insert the footer directly after the loader. |
 | [rules.json](rules.json) | Describe how url/metric/txn-name rules should be applied. |
 | [rum_client_config.json](rum_client_config.json) | These tests dictate the format and contents of the browser monitoring client configuration.  For more information see: [SPEC](https://newrelic.atlassian.net/wiki/display/eng/BAM+Agent+Auto-Instrumentation) |
-| [sql_parsing.json](sql_parsing.json) | These tests show how an SQL string should be parsed for the operation and table name. |
+| [sql_parsing.json](sql_parsing.json) | These tests show how an SQL string should be parsed for the operation and table name. *Java Note*: The Java Agent is [out-of-sync with these tests](https://source.datanerd.us/java-agent/java_agent/blob/master/newrelic-agent/src/main/java/com/newrelic/agent/database/DefaultDatabaseStatementParser.java), [has its own tests](https://source.datanerd.us/java-agent/java_agent/blob/master/newrelic-agent/src/test/java/com/newrelic/agent/database/DatabaseStatementResponseParserTest.java), and cannot implement these without a breaking change. |
```

Comment:
This is an interesting inclusion related to our discussion earlier about the attributes tests. I wonder if we should add something similar for the places where Ruby is out-of-sync with the tests?

Java may also have advice about how they avoid overwriting their unique changes.

Reply:
I like this. I've created #1152 to track it.